server/paths/manage.ts

```diff
@@ -66,7 +66,9 @@ const paths = {
   lostBeds: {
     new: lostBedsPath.path('new'),
     create: lostBedsPath,
+    index: singlePremisesPath.path('lost-beds'),
```

Comment:
i'm a bit unsure about what to put the path under, as the `lostBedsPath` includes a `bed` id, which seems unnecessary - the lost beds 'belong' to a single premises?

Reply:
I think it'll just be `lostBedsPath`, which will equate to `/premises/$premisesId/lost-beds`

Reply:
the `lostBedsPath` is actually `/premises/$premisesId/beds/:bedId/lost-beds` I think? so I'd have to pass the unnecessary bed id along, whereas it seems like `/premises/$premisesId/lost-beds` because the lost-beds belong to the premises rather than beds...

Reply:
Oh sorry, you're right! I was looking in the wrong place! In that case, yeah, let's leave it as is.